sycl/include/sycl/ext/oneapi/matrix/matrix.hpp

```diff
@@ -7,7 +7,7 @@
 // ===--------------------------------------------------------------------=== //
 /// Currently, this is the compilation command line needed to invoke AMX unit of
 /// Sapphire Rapids CPU: clang++ -fsycl -march=sapphirerapids
-/// fsycl-targets="spir64_x86_64-uknown-linux-sycldevice" -O2 main.cpp
+/// fsycl-targets="spir64_x86_64-uknown-linux" -O2 main.cpp
```

Comment:
```suggestion
/// fsycl-targets="spir64_x86_64-unknown-linux" -O2 main.cpp
```
nit: typo was already existed

Reply:
Fixed by https://github.com/intel/llvm/pull/3929/commits/21aabefa25a259224699815707d880371ab4a70f.